modules/samples/artifacts/JmsSample/jms-sample.siddhi

```diff
@@ -0,0 +1,71 @@
+@App:name('jms-sample')
```

Comment:
this sample name should be aligned with other transports eg: ReceiveHTTPInJsonFormatWithCustomMapping